pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml

```diff
@@ -1,10 +1,14 @@
 - use_sign_up_text ||= false
 - link_suffix ||= nil
+- style ||= nil
 - data = {}
 - data[:useSignUpText] = use_sign_up_text
 - data[:nominated] = params[:nominated]
 - data[:linkSuffix] = link_suffix
+- data[:style] = style
 
-%script{src: webpack_asset_path('js/code.org/views/professional_learning_apply_banner.js'), data: {props: data.to_json}}
+-# Professional Learning is only available in the US
+- if I18n.locale == :"en-US" && DCDO.get("teacher_applications_2021", CDO.default_teacher_applications_2021)
```

Comment:
Interesting, TIL you can have symbols with dashes in them :) I see most of the testing looks like it was done in en-US, were you able to confirm that banners were missing in the appropriate places for non-US?

Reply:
You're right, I didn't add many tests... let me see if I can figure that out!

Reply:
Sounds good! To be clear I'm not suggesting you necessarily need to add a bunch of tests but at least confirm that the banner isn't showing up in non-en-US, which you may have already done but just didn't make for a great screenshot :).